documentation/src/main/markdown/releasenotes.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 8.7.0
 
+### New features
+
+* (IDETECT-3655) The accuracy, HIGH or LOW, of any detectors run during a scan will now be recorded in the status.json file.
```

Comment:
One suggested tweak, just to add a small detail to the entry:
(IDETECT-3655) The accuracy of dependency determination, HIGH or LOW, of any detectors run during a scan will now be recorded in the status.json file.